mage_ai/data_preparation/models/pipeline.py

```diff
@@ -128,7 +128,8 @@ def __init__(
 
         self.repo_path = repo_path or get_repo_path()
         self.retry_config = {}
-        self.run_pipeline_in_one_process = False
+        rpiop = os.getenv('RUN_PIPELINE_IN_ONE_PROCESS', 'False') == 'True'
```

Comment:
I would put this in settings/server.py import RUN_PIPELINE_IN_ONE_PROCESS in this pipeline.py